packages/react/src/shared.tsx

```diff
@@ -115,7 +116,7 @@ export function createSharedDependencies(config: LensConfig, { onLogout, onError
     [TransactionKind.APPROVE_MODULE]: new NoopResponder(),
     [TransactionKind.COLLECT_PUBLICATION]: new NoopResponder(),
     [TransactionKind.CREATE_COMMENT]: new NoopResponder(),
-    [TransactionKind.CREATE_POST]: new NoopResponder(),
+    [TransactionKind.CREATE_POST]: new CreatePostResponder(apolloClient),
```

Comment:
Should we follow the same flow for comments to add comments to the feed?

Reply:
I don't think so, maybe we can consider adding comments to relevant comments query... although matching the cached edge could be tricky without assuming some restriction to the `publication` query. Maybe the fact we have a `useComments` dedicated hook which encapsulate the query request details can help in that sense. Thoughts?

Reply:
not sure about touching yet relevant comments query, let's see later from the consumer's side if they see it's required there.
What we can do right now is to just update `profile` comments list.
Does that make sense?

Reply:
I don't think have a dedicated hook for a profile comments.... in the end it was a way to call the `publications` query... is that you are referring to with?
> update profile comments list

Reply:
I see, now I get the problem. 
Not sure if the dedicated hook would change the complexity as it would not affect how you need to manually match the edge in the cache. 
Anyway we have a custom `keyArgs` configuration for the `publication` query which can be used to properly match the cached edge, right?